conda/cli/main_init.py

```diff
@@ -36,6 +36,11 @@ def configure_parser(sub_parsers: _SubParsersAction, **kwargs) -> ArgumentParser
         They're therefore implemented in ways specific to each shell. Each shell must be configured
         to make use of them.
 
+        The --condabin option adds the "$CONDA_PREFIX/condabin" directory to the PATH environment variable.
+        This directory only contains the conda executable and will not pollute your PATH so much.
+        On most shells, a small snippet is added to the shell profile. For CMD, the PATH environment
+        variable is modified directly in the registry.
```

Comment:
```suggestion
        The --condabin option adds the "$CONDA_PREFIX/condabin" directory to the PATH environment variable.
        This directory only contains the conda executable and does not contain other executables from other
        programs or libraries installed in the base conda environment. On most shells, a small snippet is added to the
        shell profile. For CMD, the PATH environment variable is modified directly in the registry.
```

@jaimergp, I was looking at this again with fresh eyes and found the phrase, "not pollute your PATH so much" to be a little ambiguous. I tried to add a succinct explanation of what you meant. Please include if you think it's an improvement.